website/src/css/custom.css

```diff
@@ -31,3 +31,25 @@
 .menu button {
   margin-bottom: 70px;
 }
+
```

Comment:
Wish I could review this, alas

Reply:
Type merging intro feels like it needs all the help it can get...

<img width="1123" alt="Screen Shot 2020-12-14 at 1 52 37 PM" src="https://user-images.githubusercontent.com/727224/102122757-f7586c00-3e13-11eb-9516-554440b4a6e5.png">